src/Microsoft.MobileBlazorBindings.Core/NativeControlComponentBase.cs

```diff
@@ -32,13 +32,20 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
                 builder.AddContent(2, childContent);
             }
 
+            int sequence = 3;
+            RenderAdditionalElementContent(builder, ref sequence);
+
             builder.CloseElement();
         }
 
         protected virtual void RenderAttributes(AttributesBuilder builder)
         {
         }
 
+        protected virtual void RenderAdditionalElementContent(RenderTreeBuilder builder, ref int sequence)
```

Comment:
I use `ref` parameter here, because I need to be able to increment `sequence` and support overriding this method.

E.g. `ItemsView` has properties `ItemTemplate` and `EmptyView`, `StructuredItemsView` inherits `ItemsView`, and has additional properties `Header` and `Footer`.

I need them to be rendered sequentially, i.e.
3 - ItemTemplate
4 - EmptyView
5 - SctructuredView
6 - ItemsView

Currently I can make it something like that:
```csharp
    public abstract class ItemsView<T> : View
    {
        ...
        protected override void RenderAdditionalElementContent(RenderTreeBuilder builder, ref int sequence)
        {
            RenderTreeBuilderHelper.AddDataTemplateProperty(builder, sequence++, typeof(ItemsView<T>), nameof(ItemTemplate), ItemTemplate);
            RenderTreeBuilderHelper.AddContentProperty(builder, sequence++, typeof(ItemsView<T>), nameof(EmptyView), EmptyView);
        }
    }


    public abstract class StructuredItemsView<T> : ItemsView<T>
    {
        ...
        protected override void RenderAdditionalElementContent(RenderTreeBuilder builder, ref int sequence)
        {
            base.RenderAdditionalElementContent(builder, ref sequence);

            RenderTreeBuilderHelper.AddContentProperty(builder, sequence++, typeof(StructuredItemsView<T>), nameof(Header), Header);
            RenderTreeBuilderHelper.AddContentProperty(builder, sequence++, typeof(StructuredItemsView<T>), nameof(Footer), Footer);
        }
    }
```

Let me know if you have better ideas!